lib/sparse/src/index/posting_list.rs

```diff
@@ -196,6 +196,11 @@ impl<'a> PostingListIterator<'a> {
         self.elements.get(self.current_index)
     }
 
+    /// Returns the next element *without* bounds checking.
+    pub fn get_unchecked(&self) -> &PostingElement {
+        unsafe { self.elements.get_unchecked(self.current_index) }
+    }
```

Comment:
Do we guarantee this is always safe?

I think we should either:
- if always safe, rename to `get()`
- if not always safe, color this function to be `unsafe` as well

Also, does this give us any measurable difference?